### ISSUE TITLE
Warning: Can't perform a React state update on an unmounted component.

### ISSUE THREAD

**Issue by vincenzopalazzo** [jacklam718/react-native-popup-dialog#257]
I receive this error when I use the BottomModal

```
Warning: Can't perform a React state update on an unmounted component. This is a no-op, but it indicates a memory leak in your application. To fix, cancel all subscriptions and asynchronous tasks in the componentWillUnmount method.
    in BaseModal (at Modal.js:73)
    in _default (at react-native-root-siblings/index.js:87)
```
I hope it can help

**Comment by jacklam718** [comment]
it seems you are using the old version. it probably has been fixed at the latest version.

**Comment by vincenzopalazzo** [comment]
Thanks for answering, with the last version I have this exception https://github.com/jacklam718/react-native-modals/issues/241 I think inside the component BottomModal is missed some login to dismiss the component.

I don't have enough experience with react native to fix it or make some test, but I add this task at my TODO list, if you don't have time maybe I can open a PR if I found any solution.

Thanks for your time and your work :)

**Comment by jacklam718** [comment]
that's weird. I see this error message `react-native-root-siblings/index.js:87` from your exception, but `react-native-root-siblings` has been removed from the recent version. so `react-native-root-siblings/index.js:87` shouldn't appear in the latest version. I think you could try:
1. `yarn remove react-native-modals`
2. `rm -rf node_modules`
3. `yarn add react-native-modals`

**Comment by vincenzopalazzo** [comment]
Yes, I solved issue #241 with the downgrade of the library version, I can't use the last version because react-native throw the exception described inside this issue #241.

So, I think that this is solved inside the last version, but I can not test it until I can not found any solution to avoid the exception  described in #241
